os/win/window.h

```diff
@@ -138,6 +138,7 @@ namespace os {
     gfx::Rect m_restoredFrame;
 #endif
 
+    KeyScancode m_prev_keyscancode = KeyScancode::kKeyNil;
```

Comment:
Could you please change the name of this member to m_pressedKeyscancode?

Reply:
Done :)